docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3.8'
+services:
+  db:
+    image: postgres:14.1-alpine
+    container_name: db
+    restart: always
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=pichu07
```

Comment:
@pichu07 Change password to a more general project specific password. Donot use any personal names and passwords.